array-stack-queue/1.c

```diff
@@ -0,0 +1,16 @@
+#include<stdio.h>
+#include<stdlib.h>
+int main()
+{
+  int *a,i,sum=0,n;
+  printf("Enter the n value:");
+  scanf("%d",&n);
+  a=(int *)malloc(sizeof(int)*n);
+  printf("Enter the array elements:");
+  for(i=0;i<n;i++)
+  {
+    scanf("%d",&a[i]);
+    sum+=a[i];
+  }
+  printf("\n%d",sum);
```

Comment:
Please set free your heap-memory with ```free(a);``` at the end of your main-function.  
In addition put a ```return 0;``` at the end of your main-function.